### PR TITLE
Feature/course 1

### PR DESCRIPTION
**Pull request by grojas123**
"Fixed the nctss_3.css for task2" Final PR of Tasks 2 .

### CODE REVIEW

File: staff/gustavo-rojas/git/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#To create a repository in the actual directory
```
Comment (elolivier) [inline staff/gustavo-rojas/git/README.md]: Add an space after # and before first letter to be consistent in all lines. Please validate every step to follow this comment.

File: staff/gustavo-rojas/git/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+git init
+# To see what files status are
+git status
+#To add files to the stagging area
+git add [The name of the file]
+# To add all the files with updates
+git add .
+#To send to the commit all the files in the stagging area
+git commit -m " The comment about the change"
+# to change a branch
```
Comment (elolivier) [inline staff/gustavo-rojas/git/README.md]: Start with capital letter to be consistent with all lines.

File: staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    top: 0;
+    height: 100%;
+    margin-bottom: 10px;
+}
+
+.menu span {
+    margin: 2em
+}
+
+.nameofthevent-container {
```
Comment (elolivier) [inline staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css]: Please use same convention for all class names, if you are going to separate each word with dash (-) use it everywhere. Review all class names.

File: staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+.nameofthevent-container p {
+    font-size: 70px
+}
+
+.actual_page .about_title {
+    visibility: collapse;
+}
+
+.actual_page .contact_title {
```
Comment (elolivier) [inline staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css]: Same code as actual_page .about_title, why two classes?

File: staff/gustavo-rojas/course-1-new-version/task1/about.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+</head>
+
+<body>
+  <div class=logo-container>
+    <img class="logo" src="images/logo.png" alt="The Logo of NorCal Time Trial Series">
+  </div>
+  <div class=nameofthevent-container>
+    <p class="nameoftheevent"> NorCal Time Trial Series</p>
+  </div>
+  <div class="horizontal-image-container">
```
Comment (elolivier) [inline staff/gustavo-rojas/course-1-new-version/task1/about.html]: What is the meaning of this div, it is iin all pages but i cannot see the image.

**Review by elolivier — CHANGES_REQUESTED** [review]

File: staff/gustavo-rojas/git/README.md (hunk 1 above)
Comment (grojas123) [reply staff/gustavo-rojas/git/README.md]: Done

File: staff/gustavo-rojas/git/README.md (hunk 2 above)
Comment (grojas123) [reply staff/gustavo-rojas/git/README.md]: Done.

File: staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css (hunk 3 above)
Comment (grojas123) [reply staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css]: Done.

File: staff/gustavo-rojas/course-1-new-version/task1/about.html (hunk 5 above)
Comment (grojas123) [reply staff/gustavo-rojas/course-1-new-version/task1/about.html:17]: Done .

File: staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css (hunk 4 above)
Comment (grojas123) [reply staff/gustavo-rojas/course-1-new-version/task1/styles/nctss_1.css]: I corrected I mean .actual-page.contact-title to be more specific .

File: staff/gustavo-rojas/course-1-new-version/task1/about.html (hunk 5 above)
Comment (grojas123) [reply staff/gustavo-rojas/course-1-new-version/task1/about.html]: Removed the division .

**Comment by grojas123** [comment]
Task1 adjusted the comments of PR. Corrected the README.md . Removed the <div class=horizontal-image-container> and changed the way how the classes are named

**Comment by grojas123** [comment]
Need to fix one error.

**Pull request closed without merging**